article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+\[Z = f_Z(X, \epsilon_Z)  \]
+
+\[Y(z) = f_Y(X, z, \epsilon_Y)  \]
+
+Now consider the case where there exists another latent confounding variable, U, which also affects both the treatment Z, as well as the outcome, Y.
+Figure \ref{fig:simple-graph-confounded} illustrates this assumption in DAG form, and the equations below are the structural equation equivalent:
+
+\[Z = f_Z(X, U, \epsilon_Z)  \]
+
+\[Y(z) = f_Y(X, U, z, \epsilon_Y)  \]
+
+\begin{figure}[h!]
+   \centering
+   \includegraphics[height=0.5\textwidth]{simple-graph-confounded.pdf}
+   \caption{Simple DAG of the causal relationships between X, Y, Z and confounder U}
+   \label{fig:simple-graph-confounded}
+\end{figure}
+
+It is clear from Figure \ref{fig:simple-graph-confounded} that conditioning on X alone does not block all the indirect paths between Z and Y, and that ommitting U from the analysis will yield biased results of the causal relationship between Z and Y.
+The structural equations also show that the ignorability assumption
+of Z does not hold if we only condition on X, and we risk obtaining biased estimates of the causal effect of Z on Y if we fail to
+account for U, sometimes referred to as a common cause.
+Even in this very simple example with only three or four variables, one can see the advantage of using a causal graph to encode assumptions.
+This advantage of DAGs becomes even more apparent in larger problems with many covariates available, and where the causal structure of the data is way more complicated, which are the types of problems typically faced by transportation demand modelers.
+
+Another benefit of DAGs is that they come with a set of testable implications, and incorporating them in any causal
+analysis adds robustness and defensibility to one's analysis.
+We discuss those implications in section 5, and illustrate how to use those tests in one's analysis.
+
+Lastly, it is important to note that the graphical approach to causality focuses
+primarily on issues of identification of causal effects, that is, given a
+directed acyclic graph (DAG) that encodes an analyst's knowledge and belief
+about the data generation process of the problem at hand, can a specific
+causal effect be identified?
+As such, we emphasize that DAGs are great tools
+for a modeler to encode their assumptions about a problem, but not necessarily
+a guide on how to estimate a causal effects of interest.
+
+\subsection{Prior Uses of Causal Graphs in Choice Modelling}
+\label{sec:choice-graphs}
+
+In our last subsection, we reviewed the basics of causal graphs: what are they and why are they useful?
+We targeted that subsection at choice modellers who are unfamiliar with these tools.
+However, readers should be aware of the history of causal graphs in choice modelling.
+Indeed, choice modellers have used causal graphs (in limited fashion) for years.
+Here are three examples to illustrate our point.
```

Comment:
My suggestion here is to prime them for these three different types of models then present them, aka tell them what you'll tell them about, then tell them about it.